### PR TITLE
Bulk component analyses POST api endpoint

### PR DESCRIPTION
**Pull request by ravsa**

### CODE REVIEW

File: bayesian/default_config.py [hunk 1]
```diff
@@ -53,3 +53,5 @@
 GEMINI_SERVER_URL = "http://{host}:{port}".format(
                      host=os.getenv('F8A_GEMINI_SERVER_SERVICE_HOST'),
                      port=os.getenv('F8A_GEMINI_SERVER_SERVICE_PORT'))
+
+COMPONENT_ANALYSES_LIMIT = 10
```
Comment (miteshvp) [inline bayesian/default_config.py]: Can we have this set via OS template? This way we can dynamically change the limits via saas-analytics PR

File: bayesian/default_config.py [hunk 2]
```diff
@@ -53,3 +53,5 @@
 GEMINI_SERVER_URL = "http://{host}:{port}".format(
                      host=os.getenv('F8A_GEMINI_SERVER_SERVICE_HOST'),
                      port=os.getenv('F8A_GEMINI_SERVER_SERVICE_PORT'))
+
+COMPONENT_ANALYSES_LIMIT = int(os.getenv('BAYESIAN_COMPONENT_TAGGED_COUNT', 10))
```
Comment (miteshvp) [inline bayesian/default_config.py]: The variable should be `COMPONENT_ANALYSES_LIMIT`

Comment (ravsa) [reply bayesian/default_config.py]: oops 😬

Comment (ravsa) [reply bayesian/default_config.py]: changed it. Thanks :)

**Comment by miteshvp** [comment]
@ravsa - please feel free to merge on Green

**Pull request merged**